G3Common/src/users/BusinessCustomer.java

```diff
@@ -2,83 +2,80 @@
 
 /**
  * 
- * @author Lior, Guzovsky.
- * Class description: 
- * This class is derived class from Customer class
- *  which defines the main attributes and functionalities
- * of a Business customer in our system.
+ * @author Lior, Guzovsky. Class description: This class is derived class from
+ *         Customer class which defines the main attributes and functionalities
+ *         of a Business customer in our system.
  * @version 29/11/2021
  */
-public class BusinessCustomer extends Customer{
-	
+public class BusinessCustomer extends Customer {
+
 	/**
 	 * Class members description:
 	 */
-	
+
 	/**
-	 * This is the name of the employer such as Intel 
-	 * and etc.
+	 * This is the name of the employer such as Intel and etc.
 	 */
 	protected String employerName;
-	
+
 	/**
-	 * Each Business customer is 
-	 * given a specific type of budget,
-	 * it can be budget per day, week and month.
+	 * Each Business customer is given a specific type of budget, it can be budget
+	 * per day, week and month.
 	 */
 	protected BudgetType budgetOfBusinessCustomer;
-	
-	
+
 	/**
 	 * This is the customers position in the company.
 	 */
 	protected String businessCustomerPosition;
-	
-	
+
 	/**
-	 * Each Business customer has a budget max
-	 * that he can use
+	 * Each Business customer has a budget max that he can use
```

Comment:
Each business customer has a budget that he cannot exceed.

Reply:
fixed